src/pages/editor/PublicationChannelTableRow.tsx

```diff
@@ -0,0 +1,40 @@
+import { Chip, styled, TableCell, TableRow } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useFetchOrganization } from '../../api/hooks/useFetchOrganization';
+import { useFetchPublisher } from '../../api/hooks/useFetchPublisher';
+import { ClaimedChannel } from '../../types/customerInstitution.types';
+import { getIdentifierFromId } from '../../utils/general-helpers';
+import { getLanguageString } from '../../utils/translation-helpers';
+
+interface ChannelClaimTableRowProps {
+  claimedChannel: ClaimedChannel;
+}
+const StyledTableCell = styled(TableCell)({
+  verticalAllign: 'top',
+});
+export const ChannelClaimTableRow = ({ claimedChannel }: ChannelClaimTableRowProps) => {
+  const { t } = useTranslation();
+  const publisher = useFetchPublisher(getIdentifierFromId(claimedChannel.channelClaim.channel));
+  const publisherName = publisher ? publisher.data?.name : '';
+
+  const organization = useFetchOrganization(claimedChannel.claimedBy.organizationId);
+  const organizationName = organization ? getLanguageString(organization.data?.labels) : '';
```

Comment:
`publisher` og `organization` er ikke det jeg ville forventa basert på navnet, da de er hele responsen til `useQuery` med alskens properties som ikke har noe med publisher og organization å gjøre.
De vil heller aldri kunne evalueres til `false`, så ternary for å finne navn vil aldri komme i else

```suggestion
  const publisherQuery = useFetchPublisher(getIdentifierFromId(claimedChannel.channelClaim.channel));
  const publisherName = publisherQuery.data?.name;

  const organizationQuery = useFetchOrganization(claimedChannel.claimedBy.organizationId);
  const organizationName = getLanguageString(organizationQuery.data?.labels);
```